repository/Seaside-WebSocket-Core.package/WAWebSocketPusher.class/instance/addSocket..st

```diff
@@ -0,0 +1,3 @@
+accessing
+addSocket: aWebSocket
+	mutex critical: [ sockets := sockets copyWith: aWebSocket ]
```

Comment:
Is the intention of using CoW here that readonly access to the sockets (eg. just iterating) can be done without a mutex?